docs/OTSubscriber.md

```diff
@@ -15,6 +15,10 @@
 
   * **subscribeToVideo** (Boolean) — Whether to subscribe video.
 
+  * **preferredResolution** (String) — Sets the preferred resolution of the subscriber's video. The format of the string is "widthxheight", where the width and height are represented in pixels. Valid values are "1280x720", "640x480", "320x240" and "160x120".
+
```

Comment:
Is 160x120 supported?

Reply:
Looks like the supported resolutions are "1280x720", "640x480" and "352x288" for [android](https://tokbox.com/developer/sdks/android/reference/com/opentok/android/Publisher.CameraCaptureResolution.html) and [iOS](https://tokbox.com/developer/sdks/ios/reference/Constants/OTCameraCaptureResolution.html)

Reply:
Thanks guys, solved :)